packages/web/src/hooks/use-app-status.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * Module dependencies.
+ */
+
+import { useCallback, useEffect, useState } from 'react';
+import { useContracts } from 'src/context/contracts';
+import fromUnixTime from 'date-fns/fromUnixTime';
+import isAfter from 'date-fns/isAfter';
+import isBefore from 'date-fns/isBefore';
+
+/**
+ * Current date.
+ */
+
+// TODO: Should come from the server.
+const currentDate = new Date().getTime();
+
+/**
+ * App state.
+ */
+
+export const appState = {
```

Comment:
don't need to be exported. And if you really need it, maybe you can export de type and use it accordingly.